### PR TITLE
Add schema changes newly added for IS-5.11.0

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

**Review by ChamathNS — APPROVED** [review]

**Comment by Buddhimah** [comment]
Tested DB Types and versions 

MySQL 8

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step6/um/h2.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE UM_TENANT ADD UM_TENANT_UUID VARCHAR(36);
```
Comment (nilasini) [inline components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step6/um/h2.sql:1]: NOT NULL missing

Comment (Buddhimah) [reply components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step6/um/h2.sql]: done

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step6/um/db2.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE UM_TENANT ADD COLUMN UM_TENANT_UUID VARCHAR(36) NOT NULL
```
Comment (nilasini) [inline components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step6/um/db2.sql:1]: We need to create a UUID and assign for the existing datas right like "UPDATE UM_TENANT SET UM_TENANT_UUID = UUID();"

**Review by nilasini — APPROVED** [review]

**Review by IsuraD — APPROVED** [review]

**Comment by Buddhimah** [comment]
Merge Pending due to length issue in indexes

**Pull request merged**